### DISCUSSION TITLE
Hello, I have a problem with login to my principal account

### DISCUSSION THREAD

**Discussion by tomvargasd** [community/community#101538] [New to GitHub] [answered]
Well, a days ago I lost my laptop and phone, I recovery my phone number but my principal account have 2FA with auth app that I lost with my phone device and restore password I save in the laptop that I lost to. I need to login with Phone OTP code but Github need 2FA code or recovery password, but I have not any.

Any ideas like contact github support or any configuration for this.

**Comment by DarrenChochian** [comment, marked as answer]
If you ever find yourself in a tough spot where you can't get into your laptop and phone, and on top of that, the 2FA app is out of reach, just send a message to GitHub support, also make sure to provide detailed information about your account, since they can guide you through the account recovery process and help you regain control of your GitHub account.

**Reply by joaopa24** [reply]
@DarrenChochian Do you really think support will help? They say they don't help if the person loses recovery methods either

**Comment by ettaboyle** [comment]
Hey there! We know 2FA may be a new process for some folks and with that comes questions. We put together a [FAQ](https://github.com/orgs/community/discussions/68240) that will help answer your concerns above. If you still have more questions, feel free to ask them here.

**Comment by sonersyln** [comment]
In order to regain access to your GitHub account, you will need to contact GitHub Support directly.
